howtos/how-to-make-a-clickable-avatar-in-a-header/views/index.ejs

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Talk JS custom header</title>
+</head>
+
+<body>
+
+    <%- include('shared'); %>
+
+        <div>
+            <div id="users-list-container">
+                <h4>Friends</h4>
+
+                <ul id="users-list">
+                    <% if (users) { %>
+                        <% users.forEach(user => { %>
+                            <li data-id="<%= user.id %>" data-name="<%= user.name %>" data-photoUrl="<%= user.photoUrl %>"
```

Comment:
It would be much better to add the click handlers here, and then you can use `<%= user.name %>` instead of `e.target.parentElement.attributes[0].value' etc

Reply:
this should be resolved now